doc/api/process.markdown

```diff
@@ -648,6 +648,7 @@ Will print something like:
       zlib: '1.2.8',
       ares: '1.10.0-DEV',
       modules: '43',
+      icu: '55.1', // only if built with i18n support
```

Comment:
"default" is slightly misleading here. Just calling `./configure` won't install ICU, but the [default] release builds you download and use will.